r/README.Rmd

```diff
@@ -16,48 +18,77 @@ knitr::opts_chunk$set(
 
 [![conda-forge](https://img.shields.io/conda/vn/conda-forge/r-arrow.svg)](https://anaconda.org/conda-forge/r-arrow)
 
-R integration with Apache Arrow.
+[Apache Arrow](https://arrow.apache.org/) is a cross-language development platform for in-memory data. It specifies a standardized language-independent columnar memory format for flat and hierarchical data, organized for efficient analytic operations on modern hardware. It also provides computational libraries and zero-copy streaming messaging and interprocess communication.
+
+The `arrow` package exposes an interface to the Arrow C++ library.
 
 ## Installation
 
-You first need to install the C++ library:
+Installing `arrow` is a two-step process: first install the Arrow C++ library, then the R package.
 
-### macOS
+### Release version
 
-On macOS, you may install using homebrew:
+The current release of Apache Arrow is 0.13.
 
-```
+On macOS, you may install the C++ library using [Homebrew](https://brew.sh/):
```

Comment:
Could you add this instruction to https://github.com/apache/arrow/blob/master/site/install.md as a separated pull request?
Then we can remove this section from this pull request.

Reply:
https://issues.apache.org/jira/browse/ARROW-5416 created. (Though I probably won't remove this from the R README even once it's added there, just for convenience.)